tests/interchain/go.mod

```diff
@@ -292,4 +291,4 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
-replace github.com/strangelove-ventures/interchaintest/v8 => github.com/hyphacoop/interchaintest/v8 v8.2.1-0.20250108172813-d45809b8d949
+replace github.com/strangelove-ventures/interchaintest/v8 => github.com/cosmos/interchaintest/v8 v8.8.2-0.20250224162529-535379c62c73
```

Comment:
@LexaMichaelides @fastfadingviolets @dasanchez 

We switched to a version of interchaintest that supports IBC v10.

Is this going to be an issue for your tests? If there's anything that you need that isn't included, you could make a quick PR to the Cosmos fork before we start testing.